src/DIPS.Xamarin.UI/Controls/Popup/Popup.cs

```diff
@@ -7,57 +8,9 @@ namespace DIPS.Xamarin.UI.Controls.Popup
     /// <summary>
     /// Used to create an Attached property to be used in a PopupBehavior
     /// </summary>
-    public class Popup
+    [Obsolete("Popup attached property is obsolete because the attached property fits better in a modality context. Please use Modality attached property instead")]
```

Comment:
I wonder if we need to Obsolete Popup. Or if we should support both as Popup is it's own component